config/initializers/ckeditor.rb

```diff
@@ -1,4 +1,58 @@
+# Use this hook to configure ckeditor
 Ckeditor.setup do |config|
-  config.assets_languages = Rails.application.config.i18n.available_locales.map{|l| l.to_s.downcase}
-  config.assets_plugins = %w[copyformatting tableselection scayt wsc]
```

Comment:
It looks like these lines were added in #1196 in order to reduce the time the asset compilation takes. Is it safe to remove them now?

Reply:
The `config.assets_plugins` line was added via #2711 in order to fix CKEditor-dependant flaky specs, is there a good reason to delete said line?

Reply:
@aitbw To be a bit more precise, it was changed to fix the flakies. Removing it completely (like this PR does) would have also fixed the flakies, but then compilation time would have been long, as mentioned in #1196.

Reply:
:thinking: how so, @javierm ? I mean, how removing that line could have fixed the flakies?

Reply:
Ohhhhhhh, that makes sense. Thanks for the concise explanation :bowing_man: